api/mapping/management/commands/create_concept.py

```diff
@@ -0,0 +1,38 @@
+from django.core.management.base import BaseCommand, CommandError
+from data.models.concept import Concept
+from mapping.models import ScanReportConcept, ScanReportField, ScanReportValue
+from mapping.services_rules import (
+    save_mapping_rules,
+    remove_mapping_rules,
+    find_existing_scan_report_concepts
+)
+from django.http import HttpRequest
+
+
+class Command(BaseCommand):
+    help = 'create rules from existing concepts found for a given table id'
+
+    def handle(self, *args, **options):
+
+        concept_id = 8507 # id for male
```

Comment:
please set this as an input argument rather than hardcoded

Reply:
Done